algorithm.go

```diff
@@ -28,9 +29,9 @@ type Algorithm string
 
 // supported digest types
 const (
-	SHA256 Algorithm = "sha256" // sha256 with hex encoding
-	SHA384 Algorithm = "sha384" // sha384 with hex encoding
-	SHA512 Algorithm = "sha512" // sha512 with hex encoding
+	SHA256 Algorithm = "sha256" // sha256 with hex encoding (without upper hex characters. i.e. /A-F/)
```

Comment:
just put `(lower case only)` on each, the terminology `ditto` probably isn't the best...

...it makes me think of this 😭 
![](https://i.giphy.com/FoauNqTIVwuIg.webp)